src/requirements_file_exercise.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+REFERENCE_TO_REQUIREMENT_FILE_KEY = '-r '
+
+
+def create_libraries_string(path_to_file: str) -> str:
+    libraries: List[str] = []
+    files_to_scan: List[str] = [path_to_file]
+    scanned_files: List[str] = []
+
+    for current_file in files_to_scan:
+        if current_file not in scanned_files:
+            text: List[str] = read_file_lines(current_file)
+            libraries = get_libraries_list(text, libraries, files_to_scan)
+            scanned_files.append(current_file)
+
+    return ' '.join(libraries)
+
+
+def get_libraries_list(lines: List[str], libraries: List[str], files_to_scan: List[str]) -> List[str]:
```

Comment:
You always scan one file in this function so is weird that you get 'file**s**_to_scan'